### PR TITLE
Feature/ow 300 mini

### PR DESCRIPTION
**Pull request by gmininberg**

### CODE REVIEW

File: examples/unity/js/UnityTransformer.ts [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+            this.videoFrameCanvas_.width,
+            this.videoFrameCanvas_.height
+        )
+
+        if (this.unityGame_) {
+            //@ts-ignore
+            var VonageUnity: any = globalThis.VonageUnity;
+            if (typeof VonageUnity === "object") {
+                for (let i = 0; i < imageData.data.length; i += 4) {
+                    this.unityGame_.Module.HEAPF32[(VonageUnity.input.array >> 2) + i] = imageData.data[i] / 255
```
Comment (joliveraortega) [inline examples/unity/js/UnityTransformer.ts:120]: Cannot we use `HEAP32` or even`HEAPU32` so we avoid floating point operations when writing and reading data to and from unity?

File: examples/unity/README.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
+# Insertable Streams Unity sample
+This sample application shows how to integrate insertable streams (video) with unity.
+Integrating this sample with **Vonage Video SDK** will allow you to publish a unity scene as your video. 
+
+
```
Comment (joliveraortega) [inline examples/unity/README.md:5]: Remove empty line above please.

File: examples/unity/README.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+# Demo
+You can see a demo of this sample running at
```
Comment (joliveraortega) [inline examples/unity/README.md:7]: Either update this paragraph or complete it.

File: examples/unity/README.md [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+
+### Prerequisite:
+ - Unity - this sample was built with Unity *2021.3.6f1* (make sure WebGL is installed)
+
+### Building Unity:
+
+ - From Unity Hub select *Open -> Add Project From Disk* and select **UnityVideoProject**
+ - Select *File->Build Settings* and select Platform **WebGL** and click **Switch Platform**
+ - Click **Build** 
+ - On the popup window *Save As* **demoBuild** (this name is a must since we use this name in the demo to load Unity)
```
Comment (joliveraortega) [inline examples/unity/README.md:19]: Please, leave a comment about the need of creating the folder when trying this in Windows.

File: examples/unity/README.md [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+	- demoBuild.wasm
+	- demoBuild.loader.js
+	- demoBuild.framework.js
+	- demoBuild.data
+	
+### Building the app:
+ - Open your terminal at: *Examples->Unity->public* run this commands
+	 - *npm install*
+	 - *npm run dev* (for local debug run) will run on *http://localhost:3000/*
+	 - *npm run build* -> *npm run preview* (for local release run) will run on *http://localhost:4173/*
```
Comment (joliveraortega) [inline examples/unity/README.md:30]: For local testing the last command is not relevant.

File: examples/unity/README.md [hunk 6]
```diff
@@ -0,0 +1,85 @@
...
+
+## Key points for Unity JS integration
+As you can see on the above section Unity for WebGL is compiling to *WebAssembly* with *emscripten* which allow us to use *emscripten* advantages. You can read more about it here *https://docs.unity3d.com/Manual/webgl-interactingwithbrowserscripting.html*
```
Comment (joliveraortega) [inline examples/unity/README.md:33]: It is better to use the Markdown syntax `[]()` for links to external documentation.

**Comment by joliveraortega** [comment]
Are we sure are not committing files we do not need? Thanks.

File: examples/unity/js/UnityTransformer.ts (hunk 1 above)
Comment (gmininberg) [reply examples/unity/js/UnityTransformer.ts:120]: https://docs.unity3d.com/ScriptReference/Color.html is taking float.... we can do conversions but its a demo.

File: examples/unity/README.md (hunk 2 above)
Comment (gmininberg) [reply examples/unity/README.md:5]: fixed

File: examples/unity/README.md (hunk 3 above)
Comment (gmininberg) [reply examples/unity/README.md:7]: fixed

File: examples/unity/README.md (hunk 4 above)
Comment (gmininberg) [reply examples/unity/README.md:19]: fixed

File: examples/unity/README.md (hunk 5 above)
Comment (gmininberg) [reply examples/unity/README.md:30]: i didnt understand what did you mean so i didnt change anything

File: examples/unity/README.md (hunk 6 above)
Comment (gmininberg) [reply examples/unity/README.md:33]: fixed

**Comment by gmininberg** [comment]
> 

i removed some files after talking with Mohanad but the rest i left

**Pull request merged**